app/search/page.tsx

```diff
@@ -0,0 +1,41 @@
+import SearchResults from 'components/layout/search/results';
+import { defaultSort, sorting } from 'lib/constants';
+import { getProducts } from 'lib/shopify';
+
+export const runtime = 'edge';
```

Comment:
Thank you for providing an amazing demo.👍

I have a question to ask you. 

Could you please explain the rationale behind choosing to use the Edge runtime in this page?